frontend/src/store/queries/comparisons/index.ts

```diff
@@ -87,3 +87,6 @@ export const {
   useGetComparisonCarsQuery,
   useUpdatePositionsMutation,
 } = comparisonsApi;
+
+export const comparison =
+  comparisonsApi.endpoints.addCarToComparison.matchRejected;
```

Comment:
Please name it more explicitly. Because `comparison` is too generic and doesn't convey what it really means